docs/openapi/oad/amphora.oad.yaml

```diff
@@ -0,0 +1,856 @@
+openapi: 3.1.0
+info:
+  title: Amphora Service API
+  description: |
+    This API allows to interact with the Amphora service. It provides functionality to store, fetch and manage secret
+    shared data as used for MPC-based distributed computation on encrypted data in the Carbyne Stack platform.
+
+    The endpoints provided by Amphora are divided into three categories:
+    - User facing: \
+      Used by clients for secret sharing, accessing, and managing secret shares and their tags (*/input-masks*,
+      */masked-input*, */secret-shares*).
+    - Intra VCP (`/intra-vcp`): \
+      Used by services running in the same Virtual Cloud Provider to access locally stored secret shares, e.g. for
+      ingesting into a computation.
+    - Inter VCP (`/inter-vcp`): \
+      Used for communication with other Virtual Cloud Providers in the same Virtual Cloud.
+  version: 0.1.0
+  license:
+    name: Apache-2.0
+    url: https://github.com/carbynestack/amphora/blob/0b1866f9fbe1619729fc16fd8f9c7af679eaf979/LICENSE
+  contact:
+    name: Carbyne Stack
+    url: https://carbynestack.io/
+tags:
+  - name: User facing
+    description: Used by clients for secret sharing, accessing, and managing secret shares and their tags.
+  - name: Inter VCP
+    description: Used for communication with other Virtual Cloud Providers in the same Virtual Cloud.
+  - name: Intra VCP
+    description: |
+      Used by services running in the same Virtual Cloud Provider to access locally stored secret shares, e.g. for
+      ingesting into a computation.
+paths:
+  /secret-shares/{secretId}:
+    delete:
+      summary: Delete SecretShare
+      description: Delete a SecretShare with a given id.
+      operationId: deleteSecretShare
+      tags:
+        - User facing
+      parameters:
+        - name: secretId
+          description: ID of the SecretShare to delete.
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        200:
+          description: OK
+        400:
+          description: Bad Request
+          content:
+            text/plain:
+              schema:
+                type: string
```

Comment:
Shouldn't there be an explanation of what the string is that is returned here (see also many other places in the spec)?

Reply:
introduced schema `ErrorMessage`